pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -294,6 +294,14 @@ type AlertmanagerSpec struct {
 	// It requires Alertmanager >= 0.27.0.
 	// +optional
 	EnableFeatures []string `json:"enableFeatures,omitempty"`
+	// AdditionalArgs allows to add some other flags besides the ones part of EnableFeatures
+	// For instance --auto-gomemlimit.ratio
+	// The ratio of reserved GOMEMLIMIT memory to the detected maximum container or system memory.
+	// The value must be greater than 0 and less than or equal to 1.
+	//
+	// It requires Alertmanager >= 0.28.0.
+	// +optional
+	AdditionalArgs string `json:"additionalArgs,omitempty"`
```

Comment:
Maybe use `array` for other fields will be added in the future

Reply:
Same here